releasenotes/notes/dd-trace-api-integration-0fa7ea051a4d6ce1.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    dd-trace-api: adds a simple and minimal instrumentation for the dd-trace-api-py package
```

Comment:
I don't think we need this, since we aren't providing support for customers for this API yet, so there isn't really anything new customer facing yet.